bin/src/learner/predict.py

```diff
@@ -9,16 +7,16 @@ class PredictWrapper():
     A wrapper to predict the output of a model on a dataset.
     It also provides the functionalities to measure the performance of the model.
     """
-    def __init__(self, model: object, data_path: str, experiment: object, split: int, batch_size: int = None, loss_dict: dict = None):
+    def __init__(self, model: object, data: object, loss_dict: dict = None):
```

Comment:
```suggestion
    def __init__(self, model: object, dataloader: object, loss_dict: dict = None):
```

Reply:
so that is clearly understandable that the input is not just raw data but e torch dataloader that already has the data in  it.